### DISCUSSION TITLE
how to run or implement a fine tuned model on local pc?

### DISCUSSION THREAD

**Discussion by IbrahimOzmen** [idiap/coqui-ai-TTS#226] [Q&A] [answered]
I have fine tuned the model using webui from this link: https://github.com/daswer123/xtts-finetune-webui. and i downloaded the fine tuned model to my local pc. what i want do is that using this fine-tuned model as backend and implement into a mobile app. the model has various files as shown in the screenshot and i have no idea how to use or run these files. anything would be appreciated.
![image](https://github.com/user-attachments/assets/40acea86-3179-4719-9701-52ad8c8a54c4)

**Comment by eginhard** [comment, marked as answer]
https://coqui-tts.readthedocs.io/en/latest/models/xtts.html#id5
